lib/tasks/indices.rake

```diff
@@ -243,4 +243,68 @@ You should run this task if the index schema has changed.
       end
     end
   end
+
+  desc "
+  Checking to see what's in mainstream that isn't in govuk
+  "
+  task :compare_govuk_and_mainstream_yo do
+    require 'gds_api/content_store'
+
+    client = Services.elasticsearch(hosts: SearchConfig.new.base_uri, timeout: 5.0)
+    publishing_api = Services.publishing_api
+    content_store = GdsApi::ContentStore.new(Plek.new.find('content-store'))
+
+    indices = {
+      mainstream: [],
+      govuk: []
+    }
+
+    indices.each do |index, ids|
+      ScrollEnumerator.new(
+        client: client,
+        search_body: {
+          fields: ["_id"],
+          query: { match_all: {} }
+        },
+        batch_size: 500,
+        index_names: index.to_s
+      ) { |hit| hit }.map do |hit|
+        id = hit['_id']
+        id.prepend('/') unless id[0] == '/'
+        ids << id
+      end
+    end
+
+    not_in_govuk = indices[:mainstream] - indices[:govuk]
```

Comment:
It'd be interesting to reverse this as well just to see if it finds anything.

Reply:
It finds `2234` to be missing, I suppose because of the formats that have only been published to `govuk` due to migration.